app/sidekiq/identity/get_ssoe_traits_by_cspid_job.rb

```diff
@@ -24,8 +24,8 @@ def perform(cache_key, credential_method, credential_id)
       response = SSOe::Service.new.get_traits(credential_method:, credential_id:, user:, address:)
 
       log_success_or_failure(cache_key, credential_method, credential_id, response)
-    rescue => e
-      log_failure("Unhandled exception: #{e.class} - #{e.message}", credential_method, credential_id, e)
+    rescue SSOe::Errors::Error => e
+      log_failure("SSOe service error: #{e.class} - #{e.message}", credential_method, credential_id, e)
```

Comment:
we need to make sure we are still rescuing the `Sidekiq::AttrPackageError`

You can just add another `rescue` below ` rescue SSOe::Errors::Error`